app/controllers/packages_controller.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
+VENDOR_ID = '123355'
```

Comment:
If we keep it hard coded I'd like to update this to be specific. Whose vendor ID is it?

Reply:
Hm, this comment on the ticket leads me to think differently: 
> Our "custid" is "apidvcorp" and "vendorid" is "123355" --> The "custid" and "vendorid" do not change --> so we can hard-code them in mod-kb-ebsco. If we want to keep the API generic, we should actually be making a call to RM API to get "vendorid" as mentioned in additional notes section below.

Reply:
I think that's since been edited in the issue.

Reply:
Sorry, @Robdel12 , I updated the ticket after thinking it through to not have it hard-coded. I assumed you would get updates, so did not bother pinging you on Slack, my bad. There is additional notes in the ticket on how to retrieve the vendor id specific to a vendor.